src/index.js

```diff
@@ -6,6 +6,8 @@ import $ from 'jquery';
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
+import './css/main-content.scss';
```

Comment:
Did we decide if these are actually needed? Or did we keep them for 'just in case'?